client/client.go

```diff
@@ -17,7 +17,16 @@ type Client struct {
 	endpoint string
 }
 
-type Toxic map[string]interface{}
+type Attributes map[string]interface{}
+
+type Toxic struct {
+	Name       string     `json:"name"`
+	Type       string     `json:"type"`
+	Stream     string     `json:"stream"`
```

Comment:
To use omniempty we need to make `Stream` an `&string` right? Because a strings zero value is `""`?